packages/plugin-session-replay-browser/test/session-replay.test.ts

```diff
@@ -289,7 +343,7 @@ describe('SessionReplayPlugin', () => {
       expect(update.mock.calls[0][1]({})).toEqual({
         123: {
           events: [mockEventString],
-          sequenceId: 1,
+          sequenceId: 2,
```

Comment:
Before, this test wasn't splitting into 2 distinct batches?

Reply:
It was split into two batches before, this actually perserves the specific behavior of this test, notice I added a manual override to the currentSequenceId on line 324, that was to evade this if case https://github.com/amplitude/Amplitude-TypeScript/blob/721a3997673d700b6bda9302b076be0f3fab7c09/packages/plugin-session-replay-browser/src/session-replay.ts#L96-L96, which is unnecessary to test for the set of behavior I'm trying to validate in this test specifically